Dockerfile_env

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu
```

Comment:
This is file is used to build the docker image that is pulled in the yaml file. I'll add a comment on top saying the same.